site/src/examples/slider/WithInput.tsx

```diff
@@ -83,7 +75,7 @@ export const SingleWithInput = () => {
 const RangeWithInput = () => {
   const bounds: [number, number] = [0, 100];
 
-  const [value, setValue] = useState<SliderValue>([20, 60]);
+  const [value, setValue] = useState<[number, number]>([20, 60]);
```

Comment:
Set the value to an invalid value with 3 digits and the input truncates the value